src/components/controls/km/km-validation-errors.vue

```diff
@@ -0,0 +1,145 @@
+          aria-label="Close"
+          class="
+            position-absolute
+            top-0
+            end-0
+            m-2
+            bg-transparent
+            border-0
+            fs-5
+            text-secondary
+          "
+        >
+          <span aria-hidden="true">&times;</span>
+        </button>
+        <strong>
+          {{ t("contains") }} {{ reports.errors.length }} {{ t("errors") }}
+        </strong>
+
+        <ul>
+          <li
+            v-for="error in reports.errors"
+            :title="error.parameters"
+            :key="error"
+          >
+            <span>{{ t(getTranslation(error)) }}</span>
+            <ul v-if="error.properties">
+              <li
+                v-for="property in error.properties"
+                :title="error.parameters"
+                :key="property"
+              >
+                <a
+                  class="text-decoration-none"
+                  rel="noopener"
+                  href="#"
+                  @click="jumpTo($event, property.property)"
+                >
+                  {{ getLabel(property.property) }}
+                </a>
+              </li>
+            </ul>
+            <span v-if="!error.properties">
+              <a
+                class="text-decoration-none"
+                rel="noopener"
+                href="#"
+                @click="jumpTo($event, error.property)"
+              >
+                {{ getLabel(error.property) }}
+              </a>
+            </span>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup>
+import { useAttrs, defineEmits, defineProps, ref, computed, onBeforeMount, inject } from "vue";
+import $ from "jquery";
+
+const props = defineProps({
+  reports: { type: Object, required: true },
+});
+const $emits = defineEmits(["onJumpTo"]);
+const hideErrors = computed({
+  get() {
+    return props.reports.hideErrors;
+  },
+  set(newValue) {
+    props.reports.hideErrors = newValue;
+  },
+});
+const container = useAttrs().container ?? "body,html";
+let getLabel;
+
+function t(lang) {
+  return lang;
+}
+function getTranslation(error) {
+  if (error?.code === null) return "unknown";
+  if (error?.code == "Error.Mandatory") return "mandatory";
+  if (error?.code == "Error.MandatoryAnyOf") return "mandatoryAnyOf";
+  if (error?.code == "Error.InvalidValue") return "invalidValue";
+  if (error?.code == "Error.InvalidProperty") return "invalidProperty";
+  if (error?.code == "Error.UnspecifiedLocale") return "unspecifiedLocale";
+  if (error?.code == "Error.UnexpectedTerm") return "unexpectedTerm";
+  if (error?.code == "Error.InvalidType") return "invalidType";
+
+  return error?.code;
+}
+
+function onButtonClose() {
+  hideErrors.value = !hideErrors.value;
+}
+
+function jumpTo(event, field) {
+  event.preventDefault;
+  event.stopPropagation();
+  $emits("onJumpTo", field);
+}
+
+onBeforeMount(() => {
+  getLabel = inject("getLabel");
+})
```

Comment:
I don't think this need to be inside `onBeforeMount`?, move code to line 111 and make is const
```javascript
const getLabel = inject("getLabel");
```

Reply:
took the inject outside the onBeforeMount